packages/std/src/testing/mock.rs

```diff
@@ -2264,4 +2320,31 @@ mod tests {
 
         assert_eq!(digit_sum(&[255, 1]), 256);
     }
+
+    #[test]
+    fn making_an_address_works() {
+        let mock_api = MockApi::default();
+
+        assert_eq!(
+            "cosmwasm1h34lmpywh4upnjdg90cjf4j70aee6z8qqfspugamjp42e4q28kqs8s7vcp",
+            mock_api.addr_make("creator").to_string()
+        );
+
+        assert_eq!(
+            "cosmwasm1uwcvgs5clswpfxhm7nyfjmaeysn6us0yvjdexn9yjkv3k7zjhp2sly4xh9",
+            mock_api.addr_make("").to_string()
+        );
+
+        let mock_api = MockApi::default().with_prefix("juno");
+        assert_eq!(
+            "juno1h34lmpywh4upnjdg90cjf4j70aee6z8qqfspugamjp42e4q28kqsksmtyp",
+            mock_api.addr_make("creator").to_string()
+        );
```

Comment:
As above, let's go with the order actual, expected

Reply:
Adjusted as suggested.